### DISCUSSION TITLE
All Echo devices showing as unavailable

### DISCUSSION THREAD

**Discussion by chesterflaps** [alandtse/alexa_media_player#2017] [Q&A]
Hi all.

I did the 'easy' install with HACS and got all my echo devices showing in HA but they all say unavailable and all entities are also unavailable.

I've attached a screenshot.

Any ideas how I can get them to be available?

All I really want to be able to do is to get all my Echoes to announce when an automation triggers.
![Screenshot 2023-09-07 201720](https://github.com/custom-components/alexa_media_player/assets/86977015/de8ad0cb-3abe-47a2-a002-efd523dca71a)

**Comment by chesterflaps** [comment]
anyone?

**Reply by aguti76** [reply]
I have the same problem. Hope in some help from the community

**Comment by chickentwo** [comment]
I installed this integration yesterday and am having a similar issue.  
Echo dot gets discovered without any issue but the echo show still shows unavailable/offline.

I only got it to reconnect after I go into Intergrations, Alexa Media Player, click the 3 dots and then Reload.  the unavailable/offline went away within a few seconds.
